sampleapp/src/main/scala/de/up/hpi/informationsystems/sampleapp/dactors/Customer.scala

```diff
@@ -109,21 +109,21 @@ object Customer {
         }
     }
 
-    def getCustomerInfo: Try[Relation] = {
+    def getCustomerInfo: Try[Relation] = Try{
       val rowCount = relations(CustomerInfo).records.get.size
       if (rowCount > 1) {
         throw InconsistentStateException(s"this relation was expected to contain at maximum 1 row, but contained $rowCount")
       }
-      Try(Relation(relations(CustomerInfo).records.get))
+      Relation(relations(CustomerInfo).records.get)
```

Comment:
you can use your new method `immutable` here

Reply:
Oups. 😅